ci/autoscaler/scripts/deploy-apps.sh

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
+# shellcheck disable=SC2086,SC2034,SC2155
+set -euo pipefail
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source "${script_dir}/vars.source.sh"
+
+pushd "${bbl_state_path}" > /dev/null
+  eval "$(bbl print-env)"
+popd > /dev/null
+
+function find_or_create_org(){
+  local org_name="$1"
+  if ! cf orgs | grep -q "${org_name}"; then
+    cf create-org "${org_name}"
+  fi
+  cf target -o "${org_name}"
+}
+
+function find_or_create_space(){
+  local space_name="$1"
+  if ! cf spaces | grep -q "${space_name}"; then
+    cf create-space "${space_name}"
+  fi
+  cf target -s "${space_name}"
+}
+
+function cf_login(){
```

Comment:
There is already [cf_login](https://github.com/cloudfoundry/app-autoscaler-release/blob/main/ci/autoscaler/scripts/common.sh#L31) defined. May be we can source that file here for such helper functions (where necessary)